system.c

```diff
@@ -176,6 +176,12 @@ uint8_t system_execute_line(char *line)
           else { report_ngc_parameters(); }
           break;
 
+        case 'P': // Set force sensitivity parameter. Avoids hastle of changing eeprom settings.
+          char_counter++;
+          if(!read_float(line, &char_counter, &parameter)) { return(STATUS_BAD_NUMBER_FORMAT); }
```

Comment:
try to avoid 1 line 'ifs'

while your at it, probably good to put a space after the f in if